lib/_emerge/DepPrioritySatisfiedRange.py

```diff
@@ -89,7 +89,13 @@ def _ignore_satisfied_buildtime_slot_op(cls, priority):
     def _ignore_runtime(cls, priority):
         if priority.__class__ is not DepPriority:
             return False
-        return bool(priority.satisfied or priority.optional or not priority.buildtime)
+        return bool(
+            (
+                (not priority.runtime_slot_op)
+                or (priority.satisfied and priority.runtime_slot_op)
+            )
+            and (priority.satisfied or priority.optional or not priority.buildtime)
+        )
```

Comment:
We could conceivably split this method into one that ignores satisfied runtime_slot_op and one that does not, providing some finer granularity for ignore_priority loops. Maybe it's not worth the trouble until the need arises, though.

Reply:
Ah, good point. I'll add a comment about it.